### ISSUE TITLE
How is this being used?

### ISSUE THREAD

**Issue by nuynait** [swift-emacs/flycheck-swift#1]
Hi, 

I am using this along with the swift-mode to get syntax check for swift.

I don't know how this can be used. I installed this package and also flycheck and swift mode. 

When I go enable flycheck in my swift file, and use flycheck-verify-setup, it gives me the following result. 
![image](https://cloud.githubusercontent.com/assets/2152195/19493952/dd03d810-954a-11e6-96fe-3eae222e62c8.png)

It seems like everything is good. The problem is that it does not give me any syntax checking error. 

Could you please give me some help?

**Comment by nuynait** [comment]
I found that there is a cmd called `flycheck-compile` and when I use this cmd, it tells me error appears at import UIKit. I suppose it doesn't have the sdk file. So I found a variable called 

`(setq flycheck-swift-sdk-path "/Applications/Xcode.app/Contents/Developer/Platforms/iPhoneOS.platform/Developer/SDKs/iPhoneOS10.0.sdk")`

I set the sdk path which is generated by `xcrun --sdk iphoneos --show-sdk-path` this cmd and then run the `flycheck-compile` it gives me some error on building the sdk itself. Here is the result.

![image](https://cloud.githubusercontent.com/assets/2152195/19494172/d1002392-954b-11e6-940c-98a23356a3dc.png)

I am new to emacs, so I don't know what flycheck-compile do to the problem. No matter what I do the flycheck still not show a single error to me.

Thank you very much for the time.

**Comment by taku0** [comment]
Thanks for reporting. I'll investigate this on weekends.

**Comment by nuynait** [comment]
Thank you. 

emacs version : 25.1.1 Mac Emacs
spacemacs version: 0.200.2
operating system version: MacOS Sierra 10.12 (16A323)

How I config? I first enabled the swift layer, but it seems not having flycheck-swift with it. So I manually installed flycheck-swift in my custom layer and that is the problem I have after the install. I tried to exclude the default swift layer and install swift mode all together in my custom layer as well and I get the same behavior.

Thank you very much for your time.
